packages/devops/scripts/lambda/helpers/secrets.py

```diff
@@ -0,0 +1,34 @@
+# Use this code snippet in your app.
+# If you need more information about configurations or implementing the sample code, visit the AWS docs:   
+# https://aws.amazon.com/developers/getting-started/python/
```

Comment:
```suggestion
```

Reply:
Haha I thought I might keep this so as to suggest that this really is the basic aws suggestion! I'll clean it 👍